test/controllers/property.controller.test.js

```diff
@@ -1078,7 +1145,7 @@ describe('Search Through Properties', () => {
             expect(res).to.have.status(200);
             expect(res.body.success).to.be.eql(true);
             expect(res.body).to.have.property('properties');
-            expect(property._id.equals(res.body.properties[0]._id)).to.be.eql(true);
+            expect(property._id.toString()).to.be.eql(res.body.properties[0]._id);
```

Comment:
Expectation should be in the expect block not the other way round

```suggestion
            expect(res.body.properties[0]._id).to.be.eql(property._id.toString());
```